### PR TITLE
PE-4848: refactor(logger)

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
Create the `ardrive_logger` package in order to reuse the same library in the internal packages.

Changes the API of the `LogExportInfo` to add the `emailSupport` property.

The remaining changes are just changing the imports.

--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/4lkv1rgaesqng

### CODE REVIEW

File: packages/ardrive_logger/LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```
Comment (karlprieb) [inline packages/ardrive_logger/LICENSE:1]: remove it

File: packages/ardrive_logger/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## 0.0.1
```
Comment (karlprieb) [inline packages/ardrive_logger/CHANGELOG.md:1]: remove

File: packages/ardrive_logger/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<!--
```
Comment (karlprieb) [inline packages/ardrive_logger/README.md:1]: remove

File: packages/ardrive_logger/test/ardrive_logger_test.dart [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import 'package:flutter_test/flutter_test.dart';
```
Comment (karlprieb) [inline packages/ardrive_logger/test/ardrive_logger_test.dart:1]: can we have some tests?

File: packages/ardrive_logger/README.md (hunk 3 above)
Comment (thiagocarvalhodev) [reply packages/ardrive_logger/README.md:1]: We may want to leave the README for further library documentation

**Review by karlprieb — APPROVED** [review]

**Pull request merged**